oak_functions/proto/invocation.proto

```diff
@@ -44,3 +44,10 @@ enum StatusCode {
   // Indicates other internal errors at the server. Similar to HTTP 500 status code.
   INTERNAL_SERVER_ERROR = 5;
 }
+
+// Represents a request sent to an Oak Functions application. It wraps a byte array that the Wasm
+// module can interpret as a request it can handle.
+message Request {
+  // The content of the request.
+  bytes content = 1;
```

Comment:
`body`? This is symmetric to `Response` above. Also perhaps move to the top of the file, usually readers expect to see requests before responses.